canonical-websites/www.ubuntu.com
# Use shared thank-you partial for iot thank you pages

## Done

* added newsletter context
* removed old newsletter-thank-you page
* hid footer from partial for sections that don’t need it

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- [iot newsletter thankyou](http://0.0.0.0:8001/internet-of-things/thank-you?product=newsletter)
- [iot thank-you](http://0.0.0.0:8001/internet-of-things/thank-you)
- [cloud thank-you](http://0.0.0.0:8001/cloud/thank-you) with ‘More about…’
